ross/bearing_seal_element.py

```diff
@@ -300,6 +300,8 @@ def patch(self, position, ax, bk_ax):
             Axes in which the plot will be drawn.
         position : tuple
             Position (z, y) in which the patch will be drawn.
+        length : float
+            minimum length of shaft elements
         Returns
```

Comment:
It looks like the axes are not being returned, so I think we should delete that from the docstring.